RecoDataProducts/inc/STMWaveformDigi.hh

```diff
@@ -0,0 +1,35 @@
+#ifndef RecoDataProducts_STMWaveformDigi_hh
+#define RecoDataProducts_STMWaveformDigi_hh
+//
+// Data product that represents the digitized waveforms coming from the STM detectors
+// This is used for both unsuppressed and zero-suppressed waveforms
+//
+
+// C++ includes
+#include <iostream>
+#include <vector>
+#include <array>
+#include <Rtypes.h>
+
+#include "Offline/DataProducts/inc/STMChannel.hh"
+
+namespace mu2e {
+
+  class STMWaveformDigi {
+  public:
+    STMWaveformDigi() : _trigTimeOffset(0), _adcs(std::vector<int16_t>()){};
+
+    STMWaveformDigi(uint32_t trigTimeOffset, std::vector<int16_t> adcs) : _trigTimeOffset(trigTimeOffset), _adcs(adcs) {};
+
+
+    uint32_t trigTimeOffset() const { return _trigTimeOffset; }
+    const std::vector<int16_t>& adcs() const { return _adcs; }
+
+  private:
+    uint32_t _trigTimeOffset; // time offset from EWT? to first ADC value [ct]
+    std::vector<int16_t> _adcs; // vector of ADC values for the waveform
```

Comment:
Will different digis within one event have different lengths of waveform data?

At nominal intensity, roughly how many digis do you expect for each of LaBr and HPGe?

Reply:
Hi Rob. For the unsuppressed waveform digis, I think they will all be the same length. For zero-suppressed waveform digis (coming in a future PR) they could be different lengths.

Based on Jijun's most recent simulations (DocDB-42969) it looks like we expect a hit rate of ~140 kHz in the LaBr, and ~25 kHz for HPGe at nominal intensity. So that would ~0.2 hits per event in the LaBr and ~0.04 hits per event in the HPGe.